instrumentation/opentelemetry/init.go

```diff
@@ -59,37 +77,80 @@ func Init(cfg *config.AgentConfig) func() {
 		log.Fatal(err)
 	}
 
+	defaultBatcher := sdktrace.NewBatchSpanProcessor(zipkinExporter)
```

Comment:
I would not call it defaultBatcher as it is the only batcher. How about just calling it batcher?

Reply:
Done. Changed the global batcher to `globalBatcher`.